idb/postgres/postgres_migrations.go

```diff
@@ -538,3 +528,56 @@ func MakeDeletedNotNullMigration(db *IndexerDb, state *MigrationState) error {
 	}
 	return sqlMigration(db, state, queries)
 }
+
+// MaxRoundAccountedMigration converts the import state.
+func MaxRoundAccountedMigration(db *IndexerDb, migrationState *MigrationState) error {
+	db.accountingLock.Lock()
+	defer db.accountingLock.Unlock()
+
+	nextMigrationState := *migrationState
+	nextMigrationState.NextMigration++
```

Comment:
Ok, I see you've handled it here instead. This is better (since Commit can fail), but it does mean we need to remember to do this for every migration.

Maybe we can find a way to auto-increment this in the future.

Reply:
Yes, finding a way to auto-increment would be nice.